p5-travel-app/src/client/index.js

```diff
@@ -5,7 +5,11 @@
 //import functions
 
 //import style
-import "./styles/index.scss";
+//import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
if you want to commit unfinished code, please add “// TODO” or “// FIX ME” on the top of it

Reply:
done in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)